apps/web/src/app/components/Popover/index.tsx

```diff
@@ -166,13 +166,23 @@ export interface PopperProps {
   strategy?: 'fixed' | 'absolute';
 }
 
-export function PopperWithoutArrow({ show, children, placement = 'auto', anchorEl, offset }: PopperProps) {
+export function PopperWithoutArrow({
+  show,
+  children,
+  placement = 'auto',
+  anchorEl,
+  offset,
+  forcePlacement,
```

Comment:
why is the forcePlacement needed?

Reply:
<img width="801" alt="image" src="https://github.com/user-attachments/assets/fa848bc7-555e-4cd3-aa94-629510d28c73">
this was happening and additionally, the dropdown was jumping up and down when typing in the search